src/historywork/BatchDownloadWork.cpp

```diff
@@ -19,21 +17,19 @@ BatchDownloadWork::BatchDownloadWork(Application& app, WorkParent& parent,
                                      CheckpointRange range,
                                      std::string const& type,
                                      TmpDir const& downloadDir)
-    : Work(app, parent,
-           fmt::format("batch-download-{:s}-{:08x}-{:08x}", type, range.first(),
-                       range.last()))
+    : BatchWork{app, parent,
```

Comment:
Stay consistent with the use of initializer list syntax. Also changing from one form to the other doesn't accomplish anything other than making rebasing more painful